### ISSUE TITLE
Equivalent parameters for lr_mult and decay_mult of convolution in caffe in mxnet

### ISSUE THREAD

**Issue by chowkamlee81** [apache/incubator-mxnet#8584]
I want to convert PSPNET written in caffe to mxnet. In caffe under convolution and batch normalization lr_mult and decay_mult parameters are there sample prototxt below
layer {
  name: "conv5_3_pool2_conv"
  type: "Convolution"
  bottom: "conv5_3_pool2"
  top: "conv5_3_pool2_conv"
  param {
    lr_mult: 10
    decay_mult: 1
  }
  convolution_param {
    num_output: 512
    kernel_size: 1
    stride: 1
    weight_filler {
      type: "msra"
    }
    bias_term: false
  }
}
layer {
  name: "conv5_3_pool2_conv/bn"
  type: "BN"
  bottom: "conv5_3_pool2_conv"
  top: "conv5_3_pool2_conv"
  param {
    lr_mult: 10
    decay_mult: 0
  }
  param {
    lr_mult: 10
    decay_mult: 0
  }
  param {
    lr_mult: 0
    decay_mult: 0
  }
  param {
    lr_mult: 0
    decay_mult: 0
  }
  bn_param {
    slope_filler {
      type: "constant"
      value: 1
    }
    bias_filler {
      type: "constant"
      value: 0
    }
frozen: true
    momentum: 0.95
  }
}
I would like to have lr_mult and decay_mult for convolution and batch normalization of caffe equivalent in mxnet. Kindly suggest

**Comment by solin319** [comment]
You can use set_lr_mult, set_wd_mult in class Optimizer.
http://mxnet.incubator.apache.org/api/python/optimization/optimization.html

**Comment by chinakook** [comment]
```
bn = mx.sym.BatchNorm(data=conv, name='%s_bn' % name, fix_gamma=False, use_global_stats=True, eps=0.0001, attr={'lr_mult': '0.1'})

**Comment by chowkamlee81** [comment]
Thanks chinakook for your kind reply. Got the one which was desired. Thanks
